package.json

```
@@ -33,6 +34,6 @@
     "*.ts"
   ],
   "engines": {
-    "node": ">=8"
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

```suggestion
    "node": ">= 16"
```

Might as well, given what would already be a breaking release